src/toil_scripts/adam_pipeline/adam_preprocessing.py

```diff
@@ -33,14 +33,15 @@
 import os
 import textwrap
 from subprocess import check_call, check_output
-
 import yaml
```

Comment:
Ah! I didn't realize `yaml` was a 3rd party import. Yes, you're correct then that there should be a space before it. Sorry for the back and forth here.